wiki/markdown/rules.md

```diff
@@ -96,7 +96,7 @@ StartEvent is "YES" then it will be the first processed event in the aggregation
 sequence. If StartEvent is "NO" then Eiffel events of this type will be used to
 append information to existing aggregated object. If no aggregated object exist
 then it will wait a certain time as defined by property
-_waitlist.collection.ttlValue_ in [application's properties](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/application.properties)
+_waitlist.collection.ttl_ in [application's properties](https://github.com/eiffel-community/eiffel-intelligence/blob/master/src/main/resources/application.properties)
```

Comment:
Same comment here about the reference to property names. Instead of linking to application.properties here - link to configuration.md (which in turn links to application.properties)

Reply:
Done